src/test.jl

```diff
@@ -41,23 +41,54 @@ function check_coherent_types(O::Type, tow::TestObjectWrapper)
 end
 
 """
-    test(::Type{<:Interface}, obj)
+    test(m::Module)
+    test(::Type{<:Interface}, m::Module)
+    test(::Type{<:Interface}, obj::Type)
 
 Test if an interface is implemented correctly for an object,
 returning `true` or `false`.
 
 If no interface type is passed, Interfaces.jl will find all the
 interfaces available and test them.
```

Comment:
Defined in the module or implemented by types from the module?

Reply:
Yeah I'll clarify the doc